.drone.star

```diff
@@ -424,14 +441,16 @@ def compiler_supports(compiler, version, cxx):
 def compilers_in_range(ranges_str):
     supported = {
         'gcc': ['12', '11', '10', '9', '8', '7', '6', '5', '4.9', '4.8'],
-        's390x-gcc': ['11'],
+        's390x-gcc': ['12'],
         'arm64-gcc': ['11'],
+        'freebsd-gcc': ['11', '10', '9', '8'],
         'clang': ['15', '14', '13', '12', '11', '10', '9', '8', '7', '6.0', '5.0', '4.0', '3.8'],
         'apple-clang': ['13.4.1'],
-        's390x-clang': ['12'],
+        's390x-clang': ['15'],
```

Comment:
We now have

```console
>>>>> APT: INSTALL clang-15 libstdc++-10-dev...
...
E: Unable to locate package clang-15
```

Unfortunately, I see it already uses ubuntu 22.04 `droneubuntu2204:multiarch` so it's the same version as the "Linux Clang 15 (Latest)" job. That's the only idea I would have to fix this. 

But maybe there's another library already using this VM and my comments are worthless.

Reply:
switched s390x back to clang 14.